spec/ddtrace/contrib/patchable_spec.rb

```diff
@@ -36,13 +36,12 @@
 
       describe '#compatible?' do
         subject(:compatible) { patchable_class.compatible? }
-        let(:expected_compatibility) { Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.0.0') ? true : false }
```

Comment:
I can't foresee a scenario where our test suite is running on an unsupported version of Ruby.